### ISSUE TITLE
Datepicker: Screenreader doesn't read the day along with the rest of the date

### ISSUE THREAD

**Issue by eveleens** [OfficeDev/office-ui-fabric-react#9925]
## Codepen demonstrating the issue
https://developer.microsoft.com/en-us/fabric#/controls/web/datepicker

## Actual behavior:
Screen reader is not reading the week day along with the week date. It reads: "20 May 2019 selected read only, column header Tuesday...."

## Expected behavior:
 It should read as "20 May 2019 Tuesday" together with the date. 

## Documentation describing expected behavior
MAS 1.3.1 - Info and Relationships

**Comment by ecraig12345** [comment]
Sorry about the delay getting to this. Just to confirm, are you using Edge+Narrator? 

@lorejoh12 Any thoughts on the expected behavior here since you've been working on the calendar/datepicker? And if we do add this, how/where?

**Comment by eveleens** [comment]
Using Edge + Narrator!

**Comment by lorejoh12** [comment]
The day of the week label is in the column header since that's where the labels themselves actually live. The information is still read, just read immediately after the date, and only if the column changes (so, if the user is on Friday Aug 2, and they arrow upwards to Friday July 26, the column didn't change so the day of the week doesn't get announced again).

This is intentional to provide some contextual clues about the table layout and appropriately group columns and rows together (rows have the week number labels if those are turned on). We could consider appending the day of the week to the date information in addition, but then that information is going to be read twice: "20 May 2019 Tuesday selected read only, column header Tuesday....".

Given that the information is there and read as intended, and gives contextual clues to the structure of the control, I see nothing in the current implementation that violates MAS 1.3.1. Please follow up if there is something specific in the requirements that I'm missing, but this is the first time I've heard this specific accessibility concern. If there is a requirement that we include the column header information in the cell itself as well, we can do that relatively easily, I'm more concerned about the information duplication.

**Comment by eveleens** [comment]
Sounds good. I'll bring this back to the accessibility team for reconsideration. Thanks Elizabeth and John!
